10up/autoshare-for-twitter
# WP.org readme.txt updates

### Description of the Change
PR adds plugin screenshots and updates wp.org readme.txt updates


### How to test the Change
Not a testable change. Just need to verify and confirm the content of readme.txt and screenshots.

### Changelog Entry
> Added - Plugin screenshots on wp.org
> Changed - Updated WP.org readme.txt


### Credits
Props @iamdharmesh 


### Checklist:
